deploy/docker/server.py

```
@@ -7,6 +7,7 @@
 """
 
 # ── stdlib & 3rd‑party imports ───────────────────────────────
+import contextlib
```

Ruff (0.11.9)

`contextlib` imported but unused

Remove unused import: `contextlib`

(F401)